### ISSUE TITLE
WSClient is not able to set up the connection

### ISSUE THREAD

**Issue by nagdahimanshu** [LiskHQ/lisk-sdk#7141]
### Expected behavior
WSClient should create a connection without any issue

### Actual behavior
Getting timeout while creating a connection

### Steps to reproduce
- Use `createWSClient` from `@liskhq/lisk-api-client`

### Which version(s) does this affect? (Environment, OS, etc...)
development

**Comment by ishantiw** [comment]
Commander start commands needs to fixed when handling api flags,
https://github.com/LiskHQ/lisk-sdk/blob/development/commander/src/bootstrapping/commands/start.ts#L180
https://github.com/LiskHQ/lisk-sdk/blob/development/commander/src/bootstrapping/commands/start.ts#L186
`mode` should be `modes` and should be an array for ex, ['ipc'] or ['ws']

Even after fixing this when we try to connect using wsClient we get the following error.

```ts
/lisk-sdk/node_modules/ws/lib/websocket.js:224
      return abortHandshake(this, this._req, msg);
             ^
Error: WebSocket was closed before the connection was established
```

**Comment by shuse2** [comment]
The RPC endpoint path was accidentally changed to `/rpc`, but it is working.
The remaining issue will be resolved together in #7277

**Comment by przemerr** [comment]
@blorgon1 this is not related, you're trying to connect to a service node:

I don't believe there's an official node with WS connection opened, here is a working code with a community node
```
const { apiClient } = require('@liskhq/lisk-client');

async function main() {
    const client = await apiClient.createWSClient('wss://mainnet-api.lemii.dev/ws');
    console.log(await client.node.getNetworkStats());
}

main()
    .catch(console.error);
``` 


if you want to connect to lisk-service you need to use socket.io library

https://lisk.com/documentation/lisk-service/index.html#the-json-rpc-api
